q2_diversity_lib/alpha.py

```diff
@@ -44,70 +44,76 @@
 
 
 # --------------------- Phylogenetic -----------------------------------------
-@_disallow_empty_tables
+@_validate_tables
 @_validate_requested_cpus
```

Comment:
```suggestion
```

Reply:
Cool thank you!

Reply:
Ok, that doesn't seem to have been it unfortunately.